okapi-test-module/ModuleDescriptor.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "test-basic-1.0.0",
+  "name": "Okapi test module",
+  "provides": [
+    {
+      "id": "test-basic",
+      "version": "2.2",
+      "handlers": [
+        {
+          "methods": [ "GET", "POST" ],
+          "pathPattern": "/testb",
+          "permissionsRequired": [ "test-basic.get.list" ]
+        }
+      ]
+    }
+  ],
+  "requires": [],
+  "launchDescriptor": {
+    "dockerImage": "okapi-test-module"
+  }
+}
```

Comment:
Looks redundant to me. There's already a descriptor in descriptors/ModuleDescriptor-template.json , target/ModuleDescriptor.json

Reply:
It is mentioned in Dockerfile and shows how to deploy a module using its docker container.